src/main.js

```diff
@@ -1,18 +1,15 @@
 import Cycle from '@cycle/core';
 import CycleWeb from '@cycle/web';
 
-import {scrollTopBehavior} from './values';
-
-import MainView from './views/main-view';
+import intent from './intent';
+import model from './models/main-model';
+import view from './views/main-view';
 
 function main({DOM}) {
-  var scrollTableNodeStream = DOM.get('#scroll-table-container', 'scroll').subscribe(
-    e => scrollTopBehavior.onNext(e.srcElement.scrollTop)
-  );
```

Comment:
An `onNext` inside a `subscribe` is an anti-pattern. `scrollTableNodeStream` is actually not a stream either, it's a subscription. onNext inside subscribe usually means you want a map. And that's exactly what we did, check the intent() function.